addons/registry-route/registry-route.addon

```diff
@@ -7,10 +7,10 @@ ssh sudo mkdir -p /var/lib/minishift/secrets
 ssh sudo chown #{user} /var/lib/minishift/secrets
 
 echo  -- Creating server cert
-!ssh /var/lib/minishift/bin/oc adm ca create-server-cert --signer-cert=/var/lib/minishift/base/openshift-apiserver/ca.crt --signer-key=/var/lib/minishift/base/openshift-apiserver/ca.key --signer-serial=/var/lib/minishift/base/openshift-apiserver/ca.serial.txt --hostnames='docker-registry-default.#{routing-suffix},docker-registry-default.#{routing-suffix}:443,docker-registry.default.svc.cluster.local,172.30.1.1' --cert=/var/lib/minishift/secrets/registry.crt --key=/var/lib/minishift/secrets/registry.key
+!ssh sudo /var/lib/minishift/bin/oc adm ca create-server-cert --signer-cert=/var/lib/minishift/base/openshift-apiserver/ca.crt --signer-key=/var/lib/minishift/base/openshift-apiserver/ca.key --signer-serial=/var/lib/minishift/base/openshift-apiserver/ca.serial.txt --hostnames='docker-registry-default.#{routing-suffix},docker-registry-default.#{routing-suffix}:443,docker-registry.default.svc.cluster.local,172.30.1.1' --cert=/var/lib/minishift/secrets/registry.crt --key=/var/lib/minishift/secrets/registry.key
```

Comment:
does the `oc` command really need to be run as root? I believe it just connects to the specified host...

Reply:
From Praveen `The command which we run using oc binary is part of the VM and not running on the host, would use the kubeconfig which is generated by `oc cluster up`. Since we are running that with sudo and that means kubeconfig owned by root and hence we need to run those commands with sudo. This is not the case when you use the oc binary of the host since here it works with login and other commands works with kubeconfig which is part of our profile`